cyberdrop_dl/main.py

```diff
@@ -254,23 +255,14 @@ async def wrapper(*args, **kwargs):
 
 
 async def scheduler(manager: Manager) -> None:
-    loop = asyncio.get_running_loop()
-
-    def shutdown(current_task: asyncio.Task):
-        log("Received keyboard interrupt, shutting down...", 30)
-        SHUTDOWN.set()
-        current_task.cancel()
-        signal.signal(signal.SIGINT, signal.SIG_DFL)
-
     for func in (runtime, post_runtime):
-        if SHUTDOWN.is_set():
+        if manager.states.SHUTTING_DOWN.is_set():
             return
-        current_task = asyncio.create_task(func(manager))
-        loop.add_signal_handler(signal.SIGINT, shutdown, current_task)
+        manager.current_task = t = asyncio.create_task(func(manager))
```

Comment:
```suggestion
        manager.current_task = task = asyncio.create_task(func(manager))
```
Rename variable `t` to `task` for better clarity.